src/extension/iSamples_rangeFacet.js

```diff
@@ -0,0 +1,146 @@
+import PropTypes from 'prop-types';
+import React from "react";
+import cx from "classnames";
+
+import RangeSlider from "./iSamples_rangeSlider";
+
+
+class iSamples_RangeFacet extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: props.value,
+      // New state values for min and max range for slider UI.
+      minValue: props.minValue,
+      maxValue: props.maxValue
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({value: nextProps.value});
+  }
+
+
+  // This function converts the faceted data to range values for the slider.  Since we have
+  // changed the functionality to no longer consult the facets when rendering, it is unneeded.
+  // facetsToRange() {
+  //   const {facets} = this.props;
+  //   if (facets == null || facets.length === 0) {
+  //     return this.state;
+  //   }
+  //   return facets
+  //     .filter((facet, i) => i % 2 === 0)
+  //     .map((v) => parseInt(v))
+  //     .sort((a, b) => a > b ? 1 : -1)
+  //     .filter((a, i, me) => i === 0 || i === me.length - 1);
+  // }
```

Comment:
This function is no longer used, but I've kept it here for reference in case we ever need to take a merge from an updated version of the source.